content/news/2020-06-01-gsoc-begins-2020.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "Google Summer of Code begins"
```

Comment:
I don't think we need two titles here. Just set this to `Ruma gets in on Google Summer of Code` and remove the heading below.